docs/security-best-practices.md

```diff
@@ -358,3 +358,71 @@ fluentd:
         mountPath: /etc/ssl/certs/
         readOnly: true
 ```
+## FIPS compliant binaries
+
+We provide FIPS compliant `OpenTelemetry Collector` binaries for your data collection and enrichment.
```

Comment:
Why OpenTelemetry Collector is in ticks?

Reply:
Good point, I've changed it to bold instead and did the same for Fluent Bit and Fluentd.